torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -129,38 +129,19 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
   }
 
   const label = context.payload.label.name;
-  if (label === "ci/master") {
-    let body = " `ci/master` label does not do anything. Did you mean `ciflow/trunk`?";
-    await context.octokit.issues.createComment(
-      context.repo({
-        body,
-        issue_number: context.payload.pull_request.number,
-      })
-    );
-    return;
-  }
   if (!isCIFlowLabel(label)) {
     return;
   }
-  const isOldCIFlowLabel =
-    label !== "ciflow/nightly" &&
-    label !== "ciflow/all" &&
-    label !== "ciflow/trunk" &&
-    label !== "ciflow/periodic" &&
-    !label.startsWith("ciflow/binaries");
-
-  if (isOldCIFlowLabel) {
+  const valid_labels = ["ciflow/pull", "ciflow/trunk", "ciflow/periodic", "ciflow/lint", "ciflow/all", "ciflow/binaries"];
+  if (!valid_labels.includes(label)) {
     let body = `We have recently simplified the CIFlow labels and \`${label}\` is no longer in use.\n`;
     body += "You can use any of the following\n";
-    body +=
-      "- `ciflow/trunk` (`.github/workflowss/trunk.yml`): all jobs we run per-commit on master\n";
-    body +=
-      "- `ciflow/periodic` (`.github/workflows/periodic.yml`): all jobs we run periodically on master\n";
+    body += "- `ciflow/pull` (`.github/workflows/pull.yml`): all jobs we run on PRs\n";
+    body += "- `ciflow/trunk` (`.github/workflows/trunk.yml`): all jobs we run per-commit on master\n";
+    body += "- `ciflow/periodic` (`.github/workflows/periodic.yml`): all jobs we run periodically on master\n";
+    body += "- `ciflow/lint` (`.github/workflows/lint.yml`): all lint jobs\n";
```

Comment:
looks good overall tho i’m not sure ciflow/lint and ciflow/pull actually work—could you confirm they are triggered on push to a tag?

Reply:
fixed, removed lint and pull, added android and some other binaries tags